CMakeLists.txt

```diff
@@ -35,12 +35,12 @@ else()
     find_package(Lua)
 endif()
 
-find_package(Boost 1.53.0 COMPONENTS system iostreams REQUIRED)
+find_package(Boost 1.58.0 COMPONENTS system iostreams REQUIRED)
 
 include(src/CMakeLists.txt)
 add_executable(tfs ${tfs_SRC})
 
-include_directories(${MYSQL_INCLUDE_DIR} ${LUA_INCLUDE_DIR} ${Boost_INCLUDE_DIRS} ${PUGIXML_INCLUDE_DIR} ${GMP_INCLUDE_DIR})
+include_directories(${MYSQL_INCLUDE_DIR} ${LUA_INCLUDE_DIR} ${Boost_INCLUDE_DIRS} ${PUGIXML_INCLUDE_DIR} ${GMP_INCLUDE_DIR} ${CMAKE_CURRENT_LIST_DIR}/ext/Beast/include/)
```

Comment:
AFAIK only the unit tests in Beast are built (because the library itself is header only). I'm not sure we need to build and run the tests when building tfs.